usort/cli.py

```diff
@@ -15,22 +15,27 @@
 from .api import usort_path, usort_stdin
 from .config import Config
 from .sorting import sortable_blocks
-from .util import TIMINGS, print_timings, try_parse
+from .types import Timing
+from .util import get_timings, print_timings, try_parse
 
 BENCHMARK = False
 
 
+def print_benchmark(timings: Sequence[Timing]) -> None:
+    if BENCHMARK:
+        print_timings(click.echo, timings)
+
+
 def usort_command(fn: Callable[..., int]) -> Callable[..., None]:
     """
     Run wrapped command, print timings if --benchmark, and exit with return code
     """
 
     @wraps(fn)
     def wrapper(*args: Any, **kwargs: Any) -> None:
-        TIMINGS.clear()
         exit_code = fn(*args, **kwargs) or 0
         if BENCHMARK:
-            print_timings(click.echo)
+            print_timings(click.echo, get_timings())
```

Comment:
With the changes necessary in each of the CLI functions, maybe get rid of the timings stuff here, and just use it for the translation of exit code? Or remove entirely?

Reply:
It's necessary here for printing timings for things that happen in the main process, like walking the requested paths, or the total end-to-end time.